src/components/MoviePages/MainPage.vue

```diff
@@ -1,38 +1,59 @@
 <template>
   <main-poster/>
+  <middleNavbar/>
   <div class="md:h-32 h-16"></div>
   <section id="trend" class=" mx-auto mt-10 container max-w-sm sm:max-w-2xl md:max-w-5xl lg:max-w-7xl scroll-mt-16">
             <div class="flex justify-between">
                 <h2 class="text-yellow-400 text-2xl">Trend movies</h2>
                 <button class="btn-secodary">more...</button>
             </div>
             <div class="w-full overflow-x-scroll whitespace-no-wrap mx-auto mt-5">
-                <div class="flex flex-row space-x-3 mx-auto [&::-webkit-scrollbar]:hidden">
+                <div class="flex flex-row space-x-3 mx-auto [&::-webkit-scrollbar]:hidden" v-if="popularMovies.length">
                     <trend-movies 
                     v-for="movie in popularMovies" 
                     :key="movie.id"
                     :rating="movie.vote_count"
                     :release-date="movie.release_date"
-                    :src="`${API_IMAGE_BASE_URL}${API_IMAGE_SIZE}${movie.poster_path}`"
+                    :src="`${API_IMAGE_BASE_URL}${API_IMAGE_SIZE_MD}${movie.poster_path}`"
                     :title="movie.original_title"
-                    :imdb="movie.vote_average"/>  
+                    :imdb="movie.vote_average"/> 
                 </div>
+                <div v-else>
```

Comment:
by this scenario if we don't have data for movies we'll watch an infinite spinner.
is it currect?